### ISSUE TITLE
Piucture fails without a reason

### ISSUE THREAD

**Issue by TrueCrashB** [ikorb/gcvideo#116]
After playing for two hours the picture fails and the issue is now all the time.
I tested it with different Tvs.
I have no glue why.
Might it be a hardware defect?
Or could there be a problems with the soldering?
But it worked fine till it fails.
![s-l1600 (1)](https://github.com/ikorb/gcvideo/assets/77795239/0e165e7b-3188-4fd1-90e0-6717c0591a1e)

**Comment by ikorb** [comment]
Have you tried to reset the settings? You can do that by holding the Start button down while entering the menu (LTrigger+RTrigger+X+Y+Start).

**Comment by TrueCrashB** [comment]
Yes, didn't work. Now there is no signal anymore. i'll check the soldering.
